.github/workflows/channels-ci.yml

```diff
@@ -171,38 +197,11 @@ jobs:
         uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
         with:
           name: webapp-artifact
-          path: mattermost-webapp.tar.gz
+          path: webapp/channels/mattermost-webapp.tar.gz
           retention-days: 14
-  upload-s3:
```

Comment:
As discussed in https://community.mattermost.com/core/pl/h5bwpfobmjddfxk7xsomoz8aoa, we're no longer going to upload S3 artifacts for mattermost-webapp builds